manifests/root_config.pp

```diff
@@ -24,62 +24,6 @@
   Optional[String] $job_hardware_install_dir               = '/home/distelli/bin',
   Optional[String] $job_hardware_agent_version             = undef,
 ) inherits cd4pe {
-  include cd4pe::anchors
 
-  # If SSL is enabled, trigger a refresh of the CD4PE service on config update.
-  # If SSL is not enabled, there's no need.
-  $notify = $ssl_enabled ? {
-    true    => Anchor['cd4pe-service-refresh'],
-    default => undef,
-  }
-
-  cd4pe_root_config { $web_ui_endpoint:
-    root_email                => $root_email,
-    root_password             => $root_password,
-    web_ui_endpoint           => $web_ui_endpoint,
-    backend_service_endpoint  => $backend_service_endpoint,
-    agent_service_endpoint    => $agent_service_endpoint,
-    storage_provider          => $storage_provider,
-    storage_endpoint          => $storage_endpoint,
-    storage_bucket            => $storage_bucket,
-    storage_prefix            => $storage_prefix,
-    s3_access_key             => $s3_access_key,
-    s3_secret_key             => $s3_secret_key,
-    artifactory_access_token  => $artifactory_access_token,
-    ssl_enabled               => $ssl_enabled,
-    ssl_server_certificate    => $ssl_server_certificate,
-    ssl_authority_certificate => $ssl_authority_certificate,
-    ssl_server_private_key    => $ssl_server_private_key,
-    ssl_endpoint              => $ssl_endpoint,
-    ssl_port                  => $ssl_port,
-    require                   => Anchor['cd4pe-service-install'],
-    notify                    => $notify,
-  }
-
-  if ($install_shared_job_hardware) {
-    warn("The Distelli agent has been deprecated in favor of using the Puppet agent for job hardware. Please migrate job hardware servers to use the Puppet agent: https://puppet.com/docs/continuous-delivery/latest/agent_migration.html")
-    $job_hardware_installed = cd4pe::has_job_hardware($web_ui_endpoint, $root_email, $root_password)
-    if(!$job_hardware_installed) {
-      $creds_hash = cd4pe::get_agent_credentials($web_ui_endpoint, $root_email, $root_password)
-
-      if($creds_hash) {
-        exec { 'wait for CD4PE':
-          require => Cd4pe_root_config[$web_ui_endpoint],
-          before => Class["pipelines::agent"],
-          command => "sleep 1",
-          path => "/usr/bin:/bin"
-        }
-
-        class { 'pipelines::agent':
-          access_token => Sensitive($creds_hash[access_token]),
-          secret_key   => Sensitive($creds_hash[secret_key]),
-          download_url => "${web_ui_endpoint}/download/client",
-          start_agent  => $job_hardware_start_agent,
-          data_dir     => $job_hardware_data_dir,
-          install_dir  => $job_hardware_install_dir,
-          version      => $job_hardware_agent_version,
-        }
-      }
-    }
-  }
+  warn('This version of puppetlabs-cd4pe no longer supports CD4PE root account configuration.')
```

Comment:
Should we mention the actual version number "4.0" somewhere in this message?

Imagine if somebody sees this message in 2 years